libafl_nyx/src/helper.rs

```diff
@@ -0,0 +1,97 @@
+/// [`NyxHelper`] is used to wrap `NyxProcess`
+use std::{
+    fmt::{self, Debug},
+    path::Path,
+};
+
+use libnyx::{NyxProcess, NyxReturnValue};
+
+pub type NyxResult<T> = Result<T, String>;
```

Comment:
hmm, I think we want this to be libafl's Error type..? instead of `String`

Reply:
And probably we don't need to do a type definition here, we use a raw `Result` everywhere else in the codebase.

Reply:
replace with `libafl::Error` now